### PR TITLE
1.0

### PR DESCRIPTION
**Pull request by amitmiran137**

### CODE REVIEW

**Comment by MatanBobi** [comment]
Also, these are breaking changes, they should cause a major version bump in the packages..

File: plugins/plugin-chart-composed/src/plugin/buildQuery.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ export default function buildQuery(formData: QueryFormData) {
       }
     }
     for (let i = 0; i < MAX_FORM_CONTROLS / 2; i++) {
-      const groupBy = formData?.group_by?.[i] as string;
+      const groupBy = formData.groupby?.[i] as string;
```
[the remark this answers is not in the document]
Comment (amitmiran137) [reply plugins/plugin-chart-composed/src/plugin/buildQuery.ts:47]: for ever and ever

File: plugins/plugin-chart-composed/src/plugin/buildQuery.ts [hunk 2]
```diff
@@ -53,6 +53,8 @@ export default function buildQuery(formData: QueryFormData) {
       {
         ...baseQueryObject,
         orderby,
+        columns: [],
```
[the remark this answers is not in the document]
Comment (simchaNielsen) [reply plugins/plugin-chart-composed/src/plugin/buildQuery.ts:56]: I think it requires by BE, that his value will be passed as array

File: plugins/plugin-chart-composed/src/plugin/transformProps.ts [hunk 3]
```diff
@@ -29,8 +29,8 @@ import {
 } from './utils';
 
 export default function transformProps(chartProps: ChartProps) {
-  const { width, height, queryData, queriesData } = chartProps;
-  const data = (queriesData?.[0]?.data || queryData?.data) as Data[];
+  const { width, height, queriesData } = chartProps;
+  const data = queriesData[0].data as Data[];
```
[the remark this answers is not in the document]
Comment (simchaNielsen) [reply plugins/plugin-chart-composed/src/plugin/transformProps.ts:33]: no, superset always returns data here

**Pull request merged**